source/CIMenuScene.h

```diff
@@ -80,7 +90,8 @@ class MenuScene : public cugl::Scene2 {
      * This constructor does not allocate any objects or start the game.
      * This allows us to use the object without a heap pointer.
      */
-    MenuScene() : cugl::Scene2(), _volume(0.0f), _musicOn(true), _parallaxOn(true) {}
+    MenuScene() : cugl::Scene2(), _volume(0.0f), _musicOn(true), _parallaxOn(true),
```

Comment:
I know this will just get overwritten in init, but can we make the initial value for volume 0.5?